client/pages/stripe.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import { loadStripe } from '@stripe/stripe-js';
+import { Elements } from '@stripe/react-stripe-js';
+
+import CheckoutForm from './checkout-form';
+import '../../server/public/stripe.css';
+
+// Make sure to call loadStripe outside of a component’s render to avoid
+// recreating the Stripe object on every render.
+// This is your test publishable API key.
+const stripePromise = loadStripe('pk_test_51Ly5zQD9hcLXyrLfzzKGfW7VlFhNh5usEvLD2b4yOHnUnobBGxJCh7sJaftQoocfuVGUJOP1XQWhrhIFbA0X53KT00X0pmD4Bs');
+
```

Comment:
We should put this in `.env` as `STRIPE_API_KEY`. Not for security purposes, but to keep the codebase flexible so that the code doesn't need to be modified to use the real key later on.

you can access it as `process.env.STRIPE_API_KEY`. You will need to modify your `webpack.config.js`

https://github.com/Learning-Fuze/final-project-instructions/blob/main/code-base.md#webpackconfigjs